deploy/helm_charts/dc_website/values.yaml

```diff
@@ -136,7 +136,11 @@ nl:
   models:
   memory: "2G"
   workers: 1
-  vertex_ai_models:
+  embeddingsSpec:
+    defaultIndex: ""
+    enabledIndexes: []
```

Comment:
given defaultIndex, shall we make this "additionalIndexes"?

Reply:
as discussed offline, leaving as is